backend/app/services/knowledge/protected_model_resolver.py

```
@@ -0,0 +1,299 @@
+    ) -> list[dict[str, Any]]:
+        """Load KB names and summary-model refs in request order."""
+        if not knowledge_base_ids:
+            return []
+
+        try:
+            knowledge_bases = (
+                db.query(Kind)
+                .filter(
+                    Kind.id.in_(knowledge_base_ids),
+                    Kind.kind == "KnowledgeBase",
+                    Kind.is_active == True,
```

Ruff (0.15.7)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)